spot-contracts/contracts/RolloverVault.sol

```diff
@@ -624,61 +716,64 @@ contract RolloverVault is
     //--------------------------------------------------------------------------
     // External & Public read methods
 
-    /// @inheritdoc IVault
-    function computeMintAmt(uint256 underlyingAmtIn) public view returns (uint256) {
-        //-----------------------------------------------------------------------------
-        uint256 feePerc = feePolicy.computeVaultMintFeePerc();
-        //-----------------------------------------------------------------------------
+    /// @notice Computes the amount of notes minted when given amount of underlying asset tokens
+    ///         are deposited into the system.
+    /// @param underlyingAmtIn The amount underlying tokens to be deposited into the vault.
+    /// @param feePerc The percentage of minted notes paid as fees.
+    /// @return noteAmtMint The amount of notes to be minted.
+    function computeMintAmt(uint256 underlyingAmtIn, uint256 feePerc) public view returns (uint256 noteAmtMint) {
+        uint256 noteSupply = totalSupply();
 
+        //-----------------------------------------------------------------------------
         // Compute mint amt
-        uint256 noteSupply = totalSupply();
-        uint256 notes = (noteSupply > 0)
+        noteAmtMint = (noteSupply > 0)
             ? noteSupply.mulDiv(underlyingAmtIn, getTVL())
             : (underlyingAmtIn * INITIAL_RATE);
 
         // The mint fees are settled by simply minting fewer vault notes.
-        notes = notes.mulDiv(ONE - feePerc, ONE);
-        return notes;
+        noteAmtMint = noteAmtMint.mulDiv(ONE - feePerc, ONE);
     }
 
-    /// @inheritdoc IVault
-    function computeRedemptionAmts(uint256 noteAmtBurnt) public view returns (TokenAmount[] memory) {
+    /// @notice Computes the amount of asset tokens redeemed when burning given number of vault notes.
+    /// @param noteAmtBurnt The amount of notes to be burnt.
+    /// @param feePerc The percentage of minted notes paid as fees.
```

Comment:
```suggestion
    /// @param feePerc The percentage of redeemed notes paid as fees.
```
But pay attention to the other comment about the semantic difference between "redeem" and "burn", so we're consistent.